odoo/tools/cloc.py

```diff
@@ -99,7 +99,7 @@ def count_path(self, path, exclude=None):
                 pass
         if not exclude:
             exclude = set()
-        for i in exclude_list:
+        for i in filter(bool, exclude_list):
```

Comment:
```suggestion
        for i in filter(None, exclude_list):
```
using `None` already remove falsy values.

Reply:
> using `None` already remove falsy values.

I don't really feel like it makes it any clear, nor does it look like it will make it significantly faster, but I'll trust you on that one.

It's been changed.